code/datums/uplink_items/uplink_general.dm

```diff
@@ -118,31 +118,34 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 		if(cost > U.uses)
 			return FALSE
 
-		var/obj/I = spawn_item(get_turf(user), U)
-
-		if(I)
-			if(ishuman(user))
-				var/mob/living/carbon/human/A = user
-				if(limited_stock > 0)
-					log_game("[key_name(user)] purchased [name]. [name] was discounted to [cost].")
-					if(!user.mind.special_role)
-						message_admins("[key_name_admin(user)] purchased [name] (discounted to [cost]), as a non antagonist.")
-
-				else
-					log_game("[key_name(user)] purchased [name].")
-					if(!user.mind.special_role)
-						message_admins("[key_name_admin(user)] purchased [name], as a non antagonist.")
+		if(limited_stock > 0) // only decrement it if it's actually limited
+			limited_stock--
 
-				A.put_in_any_hand_if_possible(I)
-
-				if(istype(I,/obj/item/storage/box/) && I.contents.len>0)
-					for(var/atom/o in I)
-						U.purchase_log += "<BIG>[bicon(o)]</BIG>"
-
-				else
-					U.purchase_log += "<BIG>[bicon(I)]</BIG>"
+		var/obj/I = spawn_item(get_turf(user), U)
 
-		return TRUE
+		if(!I)
+			return TRUE
+		if(limited_stock > 0)
+			log_game("[key_name(user)] purchased [name]. [name] was discounted to [cost].")
+			if(!user.mind.special_role)
+				message_admins("[key_name_admin(user)] purchased [name] (discounted to [cost]), as a non antagonist.")
+
+		else
+			log_game("[key_name(user)] purchased [name].")
+			if(!user.mind.special_role)
+				message_admins("[key_name_admin(user)] purchased [name], as a non antagonist.")
+
+		if(istype(I, /obj/item/storage/box) && length(I.contents))
+			for(var/atom/o in I)
+				U.purchase_log += "<BIG>[bicon(o)]</BIG>"
```

Comment:
Span class? Seems like it should be lowercase

Reply:
but have you considered, <BIG> /s

(this is just a relic of old code, i was moving stuff around)